sql/executor.go

```diff
@@ -41,8 +41,9 @@ var errTransactionInProgress = errors.New("there is already a transaction in pro
 
 // An Executor executes SQL statements.
 type Executor struct {
-	db     client.DB
-	nodeID uint32
+	db      client.DB
+	nodeID  uint32
+	reCache *parser.RegexpCache
```

Comment:
Yeah, maybe this is fine as is. It doesn't sit right, but that's probably the correct thing in Go.